### PR TITLE
Add algorithms docs

### PR DESCRIPTION
**Pull request by lordoz234**
Added missing docs in algorithms

### CODE REVIEW

File: daal4py/sklearn/linear_model/_coordinate_descent.py [hunk 1]
```diff
@@ -662,10 +734,82 @@ def __init__(
...
+
+            y : {ndarray, sparse matrix} of shape (n_samples,) or \
+                (n_samples, n_targets)
+                Target. Will be cast to X's dtype if necessary.
+
+            sample_weight : float or array-like of shape (n_samples,), default=None
+                Sample weights. Internally, the `sample_weight` vector will be
+                rescaled to sum to `n_samples`.
+
+                .. versionadded:: 0.23
```
Comment (Alexsandruss) [inline daal4py/sklearn/linear_model/_coordinate_descent.py:793]: Check docs for version < 0.23

Comment (lordoz234) [reply daal4py/sklearn/linear_model/_coordinate_descent.py]: Fixed

**Review by Alexsandruss — APPROVED** [review]
For future: add automatic docs receiving

**Review by Pahandrovich — APPROVED** [review]

**Comment by Pahandrovich** [comment]
@Mergifyio backport rls/2021.5-rls

**Pull request merged**